src/vendor_attribute.h

```diff
@@ -0,0 +1,25 @@
+#ifndef VENDOR_ATTRIBUTE_H
+#define VENDOR_ATTRIBUTE_H
+
+#include <string>
+#include <vector>
+#include <cstdint> //uint8_t, uint32_t
+
+class VendorSpecific
+{
+    public:
+        VendorSpecific(uint8_t type, const uint8_t* data, size_t size);
+        VendorSpecific(uint8_t type, uint32_t vendorId, uint8_t vendorType, const std::vector<uint8_t>& vendorValue);
+        std::string name() const;
+        uint8_t type() const;
+        std::string value() const;
+        uint8_t vendorType() const;
+        std::string vendorId() const;
+        std::vector<uint8_t> toVector() const;
+    private:
+        uint8_t m_type;
```

Comment:
`type` is always `26`, `VENDOR_SPECIFIC`. Why do you need this method (and the member that stores the value)?